apps/explorer/src/components/placeholder/Table.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import TableCard from '../../components/table/TableCard';
+import placeholdertheme from '../../styles/placeholder.module.css';
+
+type DataType = {
+    rowCount: number;
+    rowHeight: string;
+    colHeadings: string[];
+    colWidths: string[];
+};
+
+export default function PlaceholderTable({
+    rowCount,
+    rowHeight,
+    colHeadings,
+    colWidths,
+}: DataType) {
+    const rowEntry = Object.fromEntries(
+        colHeadings.map((header, index) => [
+            `a${index}`,
+            <div
```

Comment:
I'd break this out into its own component within `ui/` so that it's easy to create additional placeholder visualizations. You'll also want to create stories for the new component as well. The component should also follow the new styling patterns instead of the css module ones.

Reply:
Implemented